### PR TITLE
Tts new branch jan5

### PR DESCRIPTION
**Pull request by TetianaTs**

### CODE REVIEW

File: test/single_tests/SS/pageobjects/ProblemCreate.page.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class ProblemCreatePage extends Page {
     }
 
     get saveButton() {
-        return $("//button[@type='submit']");
```
Comment (NatalyaTen) [inline test/single_tests/SS/pageobjects/ProblemCreate.page.js:33]: please re-store the previous selector

File: test/single_tests/SS/pageobjects/ProblemCreate.page.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ class ProblemCreatePage extends Page {
...
+        return $("ru");
```
Comment (NatalyaTen) [inline test/single_tests/SS/pageobjects/ProblemCreate.page.js:33]: ref previous line, remove this line

File: test/single_tests/TTS/pageobjects/PublicationEdit.page.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    get saveButton() {
+        return $('button[type="submit"]');
+    }
+
+    get cancelButton() {
+        return $('button.mr-3');
+    }
+
+    // async createPublication(title, image, description, content) {
```
Comment (NatalyaTen) [inline test/single_tests/TTS/pageobjects/PublicationEdit.page.js:32]: delete commented code (lines 31-42)

**Review by NatalyaTen — CHANGES_REQUESTED** [review]
please fix the code according comments

File: test/single_tests/TTS/pageobjects/PublicationEdit.page.js (hunk 3 above)
Comment (TetianaTs) [reply test/single_tests/TTS/pageobjects/PublicationEdit.page.js:32]: ok

**Review by NatalyaTen — APPROVED** [review]
looks good

**Review by NatalyaTen — APPROVED** [review]

**Review by svshml — APPROVED** [review]
lgfm

**Pull request merged**